api/lua/README.md

```diff
@@ -0,0 +1,17 @@
+# Lua API for PlayDate
+
+High-level lua API built on-top of [playdate-sys][].
```

Comment:
```suggestion
High-level Lua API built on-top of [playdate-sys][].
```

Reply:
If this suggestion is for me, I can't "edit your PR", so it's for you while I haven't permissions for it.